docs/tutorial/part-zero/index.md

```diff
@@ -1,23 +1,144 @@
+
+npm has three distinct components: the npm website, the npm registry, and the npm CLI (command line interface).
+
+- On the npm website, you can browse what JavaScript packages are available in the npm registry.
+- The npm registry is a large database of information about JavaScript packages available on npm. 
+- Once we’ve identified a package we want, we use the npm CLI to download it to our project. The npm CLI is what talks to the registry — we generally only interact with the npm website or the npm CLI.
+
+> 💡 Check out npm’s introduction, “[What is npm?](https://docs.npmjs.com/getting-started/what-is-npm)”.
+
+## Gatsby CLI
+
+The Gatsby CLI tool lets you quickly create new Gatsby-powered sites, and also run commands for developing Gatsby sites. It is a published npm package. We’ll download it from npm registry using the npm CLI.
+
+### ✋ Install the Gatsby CLI tool
+1. Navigate to the terminal.
+2. Run `npm install --global gatsby-cli`.
+
+A couple of different things are happening here. 
+
+- `npm install`:
+    - We’re using the npm CLI to install the Gatsby CLI. We install npm packages by typing `npm install`, and then the exact name that corresponds to the name of the package we want to install from the npm registry. 
+- `--global`:
+    - When we install npm packages, we can install them globally, or in a specific project. (We’ll learn about the latter, later). The `--global` flag signals that we want the first option, to install globally. This means our package will be available to us on our computer, outside of the context of a specific project.
+- `gatsby-cli`:
+    - This is the exact name our package is registered with on the [npm registry](https://www.npmjs.com/package/gatsby-cli).
+
+### ✋ Check your Gatsby CLI installation
+1. Open up your terminal. 
+2. Run `gatsby --version`.
+3. Run `gatsby --help`.
+
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_8FB0E402DCBE189CA26AEC8EB1B27767B46B544EF50AE1038B4299BF5C07AD38_1520482614025_gatsby-cli.gif)
+
+
+As before (when we checked our Node.js install), the `gatsby --version` should return a version number. `--help` shows different commands available to you, now that the gatsby-cli is installed.
+
+## Generate a new site with “Hello World”
+
+Now let’s use the gatsby-cli tool to create our first Gatsby site. The Gatsby CLI tool lets you install “starters”, which are partially built sites with some default configuration to help you get moving faster on creating a certain type of site. Let’s scaffold out our first site!
+
+### ✋ Generate a new Gatsby site
+1. Open up your terminal. 
+2. Run `gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world`. (*Note: Depending on your download speed, the amount of time this takes could vary*).
```

Comment:
It's nice to bracket the amount of time so they'll have some expectation e.g. 1-3 minutes

Reply:
Great idea!